swift/Sources/CoinType.swift

```diff
@@ -38,6 +38,7 @@ public enum CoinType: UInt32, CaseIterable {
     case zcash = 133
     case zcoin = 136
     case theta = 500
+    case lux = 3003
```

Comment:
It's not registered: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
[https://github.com/satoshilabs/slips/pull/579](url)